lib/Udp/V5MiLightUdpServer.cpp

```diff
@@ -19,12 +19,29 @@ void V5MiLightUdpServer::handleCommand(uint8_t command, uint8_t commandArg) {
     client->prepare(&FUT096Config, deviceId, groupId);
     client->updateStatus(status);
 
+    // Just some debugging stuff
+    //Serial.print("MiLight-RGBW Command received: Device ");
+    //Serial.print(deviceId);
+    //Serial.print(" - Group ");
+    //Serial.print(groupId);
+    //Serial.print(" - Command ");
+    //Serial.println(command);
+
     this->lastGroup = groupId;
   // Command set_white for RGBW
-  } else if (command >= UDP_RGBW_GROUP_ALL_WHITE && command <= UDP_RGBW_GROUP_4_WHITE) {
+} else if (command == UDP_RGBW_GROUP_ALL_WHITE || command == UDP_RGBW_GROUP_1_WHITE || command == UDP_RGBW_GROUP_2_WHITE || command == UDP_RGBW_GROUP_3_WHITE || command == UDP_RGBW_GROUP_4_WHITE) {
     const uint8_t groupId = (command - UDP_RGBW_GROUP_ALL_WHITE)/2;
     client->prepare(&FUT096Config, deviceId, groupId);
     client->updateColorWhite();
+
+    // Just some debugging stuff
```

Comment:
Maybe good to either remove the debugging or:

* Wrap it in the appropriate compiler flags (`DEBUG_PRINTF` for RF stuff, `MILIGHT_UDP_DEBUG` for UDP stuff).
* Use the `F` macro for string constants.  Think I saw you using this in one of your early revisions.

Reply:
Saw the macro but did not get its intention. Removed the stuff. It's not that important. Just using this at the beginning to get a feeling how the code works. :)

Reply:
`F` puts the string in flash memory so that they don't occupy RAM.  Not super important if the statements are opt-in via a compiler flag.